content/en/news/support/announcing-1.9-eol/index.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Support for Istio 1.9 ends on August 18th, 2021
+title: Support for Istio 1.9 ends on August 24th, 2021
 subtitle: Support Announcement
 description: Upcoming Istio 1.9 end of life announcement.
 publishdate: 2021-07-21
 aliases:
     - /news/2021/announcing-1.9-eol
 ---
 
-According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), minor releases like 1.9 are supported for three months after the next minor release. Since [1.10 was released on May 18th](/news/releases/1.10.x/announcing-1.10/), support for 1.9 will end on August 18th, 2021.
+According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), minor releases like 1.9 are supported for three months after the next minor release. Since [1.10 was released on May 18th](/news/releases/1.10.x/announcing-1.10/), support for 1.9 will end on August 24th, 2021.
```

Comment:
"Since" no longer makes sense as the dates aren't 3 months apart.  Perhaps either "1.10 was released on X, and support for 1.09 will end on X", or an edit to the page saying the date was changed?